download.md

```diff
@@ -10,7 +10,8 @@ nav_order: 4
 These binaries are all you need to get started signing chain-trees and running
 your own local test net.
 
-The most recent release (0.0.9) includes several bug fixes.
+The most recent release (0.0.10) includes several bug fixes related to changing
+ownership and importing/exporting chain-trees.
```

Comment:
Not a big deal but the "changing ownership" bug ended up being much broader than that. Any addition of new top-level keys (the first time you change owners or establish a coin does this implicitly) used to drop any existing ones.